nautobot/wireless/models.py

```diff
@@ -121,20 +121,25 @@ class RadioProfile(PrimaryModel):
     frequency = models.CharField(
         max_length=CHARFIELD_MAX_LENGTH,
         choices=RadioProfileFrequencyChoices,
+        blank=True,
     )
-    tx_power_min = models.IntegerField()
-    tx_power_max = models.IntegerField()
+    tx_power_min = models.IntegerField(blank=True, null=True)
+    tx_power_max = models.IntegerField(blank=True, null=True)
     channel_width = JSONArrayField(
-        base_field=models.IntegerField(choices=RadioProfileChannelWidthChoices), default=list
+        base_field=models.IntegerField(choices=RadioProfileChannelWidthChoices),
+        default=list,
+        blank=True,
+    )
+    allowed_channel_list = JSONArrayField(
+        base_field=models.IntegerField(blank=True, null=True), default=list, blank=True, null=True
```

Comment:
Feels odd to me that the inner `IntegerField` is also `blank=True, null=True` - is that necessary, and what does it do for us?

Also, why is `channel_width` blank=True, null=False but `allowed_channel_list` blank=True, null=True?

Reply:
I am not sure it does anything, I was having issues though with api tests till I added it. I will do some research.